Content.Client/Atmos/EntitySystems/AtmosDebugOverlaySystem.cs

```diff
@@ -42,8 +42,10 @@ public override void Initialize()
             SubscribeLocalEvent<GridRemovalEvent>(OnGridRemoved);
 
             var overlayManager = IoCManager.Resolve<IOverlayManager>();
-            if(!overlayManager.HasOverlay<AtmosDebugOverlay>())
-                overlayManager.AddOverlay(new AtmosDebugOverlay());
+            if(!overlayManager.HasOverlay<AtmosDebugWorldspaceOverlay>())
+                overlayManager.AddOverlay(new AtmosDebugWorldspaceOverlay());
+            if(!overlayManager.HasOverlay<AtmosDebugScreenspaceOverlay>())
+                overlayManager.AddOverlay(new AtmosDebugScreenspaceOverlay());
```

Comment:
You can have multiple viewing spaces in one overlay so more is redundant, also the hasoverlay check isn't needed anymore as it doesn't throw.

Reply:
Oh, cool, ok. Didn't notice you could do that. Thanks for the review.